core/ui-common/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt

```diff
@@ -445,3 +299,6 @@ data class WireDialogButtonProperties(
     val type: WireDialogButtonType = WireDialogButtonType.Secondary,
     val loading: Boolean = false
 )
+
+// todo, replace when markdown is moved to commons
```

Comment:
I left a reminder for us, so when we move Markdown components, we can replace this constant from there.